package.json

```diff
@@ -29,16 +29,16 @@
     "@babel/eslint-plugin": "",
     "@babel/preset-env": "",
     "chai": "3.4.1",
-    "es6-promise": "3.0.2",
+    "es6-promise": "4.2.4",
     "eslint": "^8.0",
-    "karma": "^6.3",
-    "mocha": "^9.0.0",
+    "eslint-plugin-jest": "^26.0.0",
+    "jest": "^27.4.7",
     "sinon": "^12.0.1",
     "webpack": "^5.0"
   },
   "dependencies": {
-    "blueimp-md5": "1.1.1",
-    "cookies-js": "1.2.2",
+    "blueimp-md5": "2.19.0",
```

Comment:
Were there any changes of note from `1.x` --> `2.x` upgrade?
I took a look at the repo https://github.com/blueimp/JavaScript-MD5 but couldn't find any release/change notes.

Reply:
I don't believe so, nothing appears to be impacted and it seems like they just started tracking versioning with 2.0.0